src/emulator/auth/operations.ts

```diff
@@ -2221,7 +2221,20 @@ async function mfaSignInFinalize(
 
   let { user, signInProvider } = parsePendingCredential(state, reqBody.mfaPendingCredential);
   const enrollment = user.mfaInfo?.find(
-    (enrollment) => enrollment.unobfuscatedPhoneInfo === phoneNumber,
+    (enrollment) => {
+        // All but firebase-ios-sdk finalize with unobfuscated phone number.
```

Comment:
linter might not be happy with this spacing nit...

```suggestion
      // All but firebase-ios-sdk finalize with unobfuscated phone number.
```